src/style.scss

```diff
@@ -76,12 +79,12 @@ input {
   position: relative;
   width: 100%;
 
-  min-height: calc(var(--auro-size-700, 56px) + 2px);
-  max-height: calc(var(--auro-size-700, 56px) + 2px);
+  min-height: calc(var(--ds-size-700, 56px) + 2px);
+  max-height: calc(var(--ds-size-700, 56px) + 2px);
 
-  padding: var(--auro-size-400, 2rem);
-  padding-right: var(--auro-size-100, .5rem);
-  padding-bottom: var(--auro-size-100, .5rem);
+  padding: var(--ds-size-400, 2rem);
+  padding-right: var(--ds-size-100, .5rem);
+  padding-bottom: var(--ds-size-100, .5rem);
```

Comment:
All these fixed fallback values should be replaced with the equivelement scss variable.

Reply:
sure, also will keep it as it is if there's none in the tokens.